mozetl/search/dashboard.py

```diff
@@ -70,6 +70,7 @@ def explode_search_counts(main_summary):
         .withColumn(exploded_col_name, explode(col('search_counts')))
         .withColumn('addon_version', udf_get_search_addon_version('active_addons'))
         .select(['*'] + search_fields)
+        .filter('single_search_count.count < 1000')
```

Comment:
Can we bump this up to 10k to be safe? IIRC, the outrageous values tend to be hundreds of millions of searches per day.

Reply:
Makes sense, I'll do that

Reply:
Drive-by comment: move this to a well-named constant which can also be used in the tests.

Reply:
Also a good idea, done